### DISCUSSION TITLE
【委托】交友（Java版MC）

### DISCUSSION THREAD

**Discussion by fulinshen** [share121/inter-knot#2433] [其他]
# 委托内容：急需一个Java版Minecraft的朋友一起玩。
## 联系方式：QQ:3756504014
![Minecraft](https://github.com/user-attachments/assets/1e20bae3-46db-451e-abf4-a13a90901a66)

**Comment by YuTinMIn** [comment]
没有正版的怎么办？也可以吗？

**Reply by fulinshen** [reply]
也可以，能关闭正版验证

**Comment by xinyuanyoo** [comment]
有正版但是不想玩了

**Comment by shuiyuan2023** [comment]
我跟你玩，我一个人也无聊

**Comment by LinkboyTvT** [comment]
跟陌生人远程联机有什么方法呢，国际版第三方启动器和正版会有区别吗

**Comment by yingguan-Cloud** [comment]
可惜了我们，我没电脑，不然就能玩了（手机上PojavLaunch上的Java版太难受了，玩不动<img src="https://bbs-static.miyoushe.com/static/2024/06/28/44e692e3ac57a9258f184be7db891f6c_2424955725600529377.png" alt="松雀-吓" width="75" >）

**Comment by JobNumber-9527** [comment]
90菲林还没我买个正版贵呢【dog】

**Comment by CloudSwordSage** [comment]
正版登不进去<img src="https://img-static.mihoyo.com/communityweb/upload/03c78415a3318d527d307c52a188a5ad.png" alt="米游姬-呆滞" width="75" >
